content/blogposts/2022/multi-version-upgrades.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Multi-version upgrades"
+description: "Placeholder."
```

Comment:
Bringing attention to this to make sure we don't forget updating this as well as the `heroImage` and `socialImage` before we merge this.

Reply:
Updated both to https://storage.googleapis.com/sourcegraph-assets/blog/multi-version-upgrades/mvu-hero.png.